src/main/java/com/mazyavr/schedule/controller/ProjectController.java

```diff
@@ -0,0 +1,44 @@
+package com.mazyavr.schedule.controller;
+
+import com.mazyavr.schedule.entity.ProjectEntity;
+import com.mazyavr.schedule.service.ProjectService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+@Controller
+@RequestMapping(path = "/projects")
+public class ProjectController {
+
+  @Autowired
+  private ProjectService projectService;
+
+  @PostMapping(path = "/add")
+  public @ResponseBody String addNewProject(@RequestParam String name) {
+    projectService.add(name);
+    return "Проект добавлен";
+  }
+
+  @DeleteMapping(path = "/delete")
+  public @ResponseBody String delProject(@RequestParam Long id) {
+    projectService.delete(id);
+    return "Проект удален";
+  }
```

Comment:
Для такого рода ручек можно возаращать просто ответ `{success: true}`. Создать где-нибудь класс

```java
public class SimpleResponse {
  public final boolean success = true;
}
```

И возвращать его

```suggestion
  public @ResponseBody SimpleResponse delProject(@RequestParam Long id) {
    projectService.delete(id);
    return new SimpleResponse();
  }
```